ui/app/templates/components/global-header.hbs

```diff
@@ -60,9 +60,27 @@
     >
       Documentation
     </a>
-    <LinkTo @route="settings.tokens" class="navbar-item"  {{keyboard-shortcut menuLevel=true pattern=(array "g" "a") }}>
-      ACL Tokens
-    </LinkTo>
+    {{#if this.token.selfToken}}
+      <PowerSelect
+        {{keyboard-shortcut menuLevel=true pattern=(array "g" "p") }}
+        @options={{this.profileOptions}}
+        @onChange={{action (queue 
+          (fn (mut this.profileSelection))
+          this.profileSelection.action
+        )}}
```

Comment:
`learning opportunity`: This is unique. Is this an instance of why you want the ergonomic benefits of `Ember 4.5`? Why are you calling the `mut` helper here? 

Are we using it for the child component `PowerSelect` to mutate the value of a parent `GlobalHeader`?

So you're creating a piped function that first mutates the value of the `profileSelection` to display to the user when the option is selected in the dropdown menu and then you're dispatching `profileSelection.action` to fire after we set the `profileSelection` value? Do I have that correct? And this is ergonomically better because now we're not writing that function in the backing component class?

Reply:
This lets us attach actions to the `profileOptions` objects themselves. PowerSelect doesn't give us a "currentlySelectedOption" entity by default in its template, so we use `profileSelection` to store that.

We don't do this to show the selection to the user; we always just show the word "Profile" to them.